src/board/internal.tsx

```diff
@@ -293,6 +293,7 @@ export function InternalBoard<D>({
                       maxHeight: gridContext.getHeight(itemMaxSize.height),
                     })}
                     onKeyMove={onItemMove}
+                    sourceId={"board"}
```

Comment:
The current assumption is that there is going to be exactly one board and at most one palette, but in order to make the solution better extensible I'd already pass the generated id here.